curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/645f764c37de33015ded1273.md

```diff
@@ -0,0 +1,305 @@
+---
+id: 645f764c37de33015ded1273
+title: Step 105
+challengeType: 0
+dashedName: step-105
+---
+
+# --description--
+
+The `user` object's property `avatar_template` is a string that contains a placeholder `{size}`. You will create a constant named `avatar` and assign it the value of the `user` object's property `avatar_template` with the placeholder `{size}` replaced with the desired size of the avatar, in this case, `30`. This will allow you to customize the size of the user's avatar in your application.
```

Comment:
It might be a bit hard to understand what exactly is asked. Maybe including a full example of the `avatar_template` string, would make it easier.